### PR TITLE
Full rewrite in SASS

### PR DESCRIPTION
**Pull request by Fuyukai**
This uses SCSS for a much more modular and configurable setup.

Still TODO:

- [ ] Ensure everything from theme.css is copied over  
- [x] Add instructions for how to turn theme.scss into theme.css  
- [ ] Some todo comments in the files  
- [ ] Try and move everything out from theme.scss into the individual modules

### CODE REVIEW

File: partials/_profile_popout.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Note: Popout means the thing when you click an avatar.
+
+// Normalise header colours
+div[class^='userPopout'] {
+    div {
+        &[class^='headerPlaying'],
+        &[class^='headerNormal'],
+        &[class^='headerSpotify'] {
+            background-color: #FF6B81;
```
Comment (Fuyukai) [inline partials/_profile_popout.scss]: Forgot to make this $main-colour

File: partials/_channel.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// CSS for inside the channel
+
+// Load more messages button
+div[class^='hasMore'] button {
+    background-color: white;
+    color: #FF6B81;
```
Comment (avayert) [inline partials/_channel.scss]: use $main-colour here?

File: partials/_channel_scroller.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                }
+                }
+
+            // change the waving icon to something else
+            svg[name='PersonWaving'] {
+                padding-bottom: 8px;
+            }
+            svg[name='PersonWaving'] path {
+                /* Material icons group */
+                d: path("M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z");
```
Comment (avayert) [inline partials/_channel_scroller.scss]: Should paths go into their own file like the background images

File: partials/_guild_scroller.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
+div[class^='guildsWrapper'] {
+    // MOST IMPORTANT CHANGE
+    // Changes the Discord icon to a House at the top of the DM list.
+    svg[name^='Discord'] defs path {
+        /* Google icons home icon */
+        d: path("M 10 20 v -6 h 4 v 6 h 5 v -8 h 3 L 12 3 L 2 12 h 3 v 8 Z") !important;
```
Comment (avayert) [inline partials/_guild_scroller.scss]: ditto

File: partials/_channel_scroller.scss (hunk 3 above)
Comment (Fuyukai) [reply partials/_channel_scroller.scss]: Considered it but they're not really long enough to justify it

**Pull request merged**